src/supabase/authProvider.js

```diff
@@ -26,11 +26,17 @@ export const supabaseAuthProvider = (
 	client,
 ) => ({
 	async login({ email, password }) {
-		const { error } = await client.auth.signInWithPassword({ email, password });
-
+		const { error, data } = await client.auth.signInWithPassword({ email, password });
+				
 		if (error) {
-				throw error;
+			throw error;
 		}
+
+		if (data.user.app_metadata.react_admin !== true) {
```

Comment:
maybe we should add this also for the other functions, like reset password, or do you have to be logged in to do that

Reply:
The functionality you are talking about does not exist. There's no way to reset your password. If you are talking about "Set Password", then yes, you need to be logged in for that.